content/data-reporting/reports-api/index.mdx

```diff
@@ -95,7 +95,59 @@ curl "https://api.livechatinc.com/v3.2/reports/chats/agents_chatting_duration?fr
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+### Tags
+
+Shows distribution of tags for chats.
+
+#### Specifics
+
+|                     |                                                       |
+| ------------------- | ----------------------------------------------------- |
+| **Method URL**      | `https://api.livechatinc.com/v3.2/reports/chats/tags` |
+| **HTTP method**     | GET                                                   |
+| **Required scopes** | -                                                     |
+
+#### Request
+
+| Query String          | Required | Notes                                                                                                                                                                   |
+| --------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                  | yes      | Date in the RFC3339 format                                                                                                                                              |
+| `from`                | yes      | Date in the RFC3339 format                                                                                                                                              |
+| `agents`              | no       | Agent emails separated by a comma; if not specified, returns the data for all agents within the license.                                                                |
+| `groups`              | no       | Group IDs separated by a comma                                                                                                                                          |
+| `names`               | no       | Names of tags separated by a comma; when tags=:without: you will get total number of chats without tags, when tags=:with: you will get total number of chats with tags. |
+| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`, `year`; defaults to `day`                                                                                         |
+
+#### Response
+
+| Field               | Notes                                                 |
+| ------------------- | ----------------------------------------------------- |
+| `total`             | The total number of chats in the specified date range |
+| `records`           | Contains `distribution` objects, for example, `day`   |
+| `records.day.<tag>` | The total number of chats tagged with `<tag>`         |
+
+</Text>
+<Code>
+<CodeSample path={'REQUEST'}>
+<Sample>
+
+```shell
+curl "https://api.livechatinc.com/v3.2/reports/chats/tags?from=2020-09-01T00:00:00%2B02:00&to=2020-09-14T23:59:59%2B02:00" \
+  -H 'Authorization: Bearer <your_access_token>'
+```
+
+</Sample>
+</CodeSample>
+
+<CodeResponse type="reports" version="v3.2" json="tags"/>
```

Comment:
You need to export it from `payloads/data-reporting/v3.2/reports-api/index.js` as it doesn't get rendered. 
![image](https://user-images.githubusercontent.com/42515529/104563651-344d9380-564a-11eb-9aea-153aa175018f.png)

Reply:
Same for v3.3